plugins/modules/route53_health_check.py

```diff
@@ -45,8 +45,19 @@
       - The type of health check that you want to create, which indicates how
         Amazon Route 53 determines whether an endpoint is healthy.
       - Once health_check is created, type can not be changed.
```

Comment:
Can you please addd a phrase in the description similar to:

`Support for C(CALCULATED) was added in release 6.3.0`

I do not think this will go into 6.2.0 giving the timing.